APISpecification.yaml

```diff
@@ -53,6 +53,63 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/GenomeDetails'
+  /api/metadata/genome/{genome_id}/karyotype:
```

Comment:
I see all endpoints here follow the `.../genome/{genome_id}` pattern. Wouldn't `.../genomes/{genome_id}` be more idiomatic?

Reply:
In general plural makes better sense in urls but here it doesn't seem like we are using /genomes by itself to retrieve a list of genomes so I'll vote for singular for this.